product_brand/models/product_brand.py

```diff
@@ -28,13 +28,14 @@ class ProductBrand(models.Model):
     )
     products_count = fields.Integer(
         string='Number of products',
-        compute='_get_products_count',
+        compute='_compute_products_count',
     )
 
-    @api.one
+    @api.multi
     @api.depends('product_ids')
-    def _get_products_count(self):
-        self.products_count = len(self.product_ids)
+    def _compute_products_count(self):
+        for this in self:
```

Comment:
FYI, we don't use too much `this` as variable name on Odoo, but `record`, `rec`, `one` or something significant according the model (`partner`, `product`...). But no need to change it here.

Reply:
Thanks, I'll keep this in mind for the next one